apis/python/src/tiledbsoma/_tdb_handles.py

```diff
@@ -369,6 +366,88 @@ def dim_names(self) -> Tuple[str, ...]:
         return tuple(self._handle.index_column_names)
 
 
+class DataFrameWrapper(SOMAArrayWrapper, Wrapper[clib.SOMADataFrame]):
+    """Wrapper around a Pybind11 SOMADataFrame handle."""
+
+    @classmethod
+    def _opener(
+        cls,
+        uri: str,
+        mode: options.OpenMode,
+        context: SOMATileDBContext,
+        timestamp: int,
+    ) -> clib.SOMADataFrame:
+        open_mode = clib.OpenMode.read if mode == "r" else clib.OpenMode.write
+        return clib.SOMADataFrame.open(
+            uri,
+            open_mode,
+            {k: str(v) for k, v in context.tiledb_config.items()},
+            [],
+            clib.ResultOrder.automatic,
+            (0, timestamp),
```

Comment:
Could you add names to these arguments? The floating `[]` makes me nervous because I have no idea what it belongs to.